cli/index.js

```diff
@@ -133,6 +134,12 @@ args.command(['d', 'docs', 'h', 'home'], 'Open the npm page of a plugin', (name,
   process.exit(0);
 });
 
+args.command(['version'], 'Show the version of hyper', () => {
+  checkConfig();
```

Comment:
This command doesn't need a valid config file.
I would remove this check.

Reply:
@chabou done.